sdk/test/UnitTests/AWSXRayRecorderTests.cs

```diff
@@ -1019,7 +1020,20 @@ public void TestInitializeInstanceWithRecorder3() // setting custom daemon addre
 #endif
             Assert.AreEqual(typeof(DummyTraceContext), AWSXRayRecorder.Instance.TraceContext.GetType()); // Custom trace context
             recorder.Dispose();
+            AWSXRayRecorder.Instance.Dispose();
+        }
+
+        [TestMethod]
+        public void TestDefaultStreamingStrategy()
+        {
+            IStreamingStrategy defaultStreamingStrategy = new DefaultStreamingStrategy(50);

```

Comment:
Lets split this unit test into separate unit tests:
1. DefaultStreamingStrategy() -> check default value is set 
2. DefaultStreamingStrategy(45) -> value other other than default value is correctly set
3. DefaultStreamingStrategy(-10) -> negative value is set